scripts/cc-cli/src/commands/distributeRewards.ts

```diff
@@ -34,6 +27,10 @@ async function distributeRewardsAction(options: OptionValues) {
     process.exit(1);
   }
 
+  const signerSeed = await getSeedFromEnvOrPrompt(
+    process.env.CC_SEED,
+    "Specify caller's seed phrase"
+  );
```

Comment:
We should probably document these 3 ENV vars. 

However we probably need to make the function a bit more generic and inspect both ENV values then return the appropriate seed depending on the command being used. 

Right now I can have 3 different seeds specified in my ENV vars and I feel like that's a bit too much.

Reply:
Well the thing is that some extrinsics like distributeRewards don't need to be called by a specific account.
I agree we should document the env vars but I didn't want to touch the README.md last night in the same PR because it's really outdated.

Reply:
I'm not sure what you mean by more generic though

Reply:
> I'm not sure what you mean by more generic though

Like cycle through the different vars. For example if any seed works then use CONTROLLER, otherwise use STASH or something like this. May not be the best UI though.

Reply:
Oh, I see what you mean.
I wouldn't do that. Everything should be very deliberate by the user.
There are definitely edge cases where a gas fee on the wrong account could mess things up for the user.

Reply:
I moved all the environment variables selection to the account file, so at least they're all in one place.

I think that's okay for documentation for now?
We'll need to rewrite the README and the help of each of the actions at some point.